### PR TITLE
OWNERS_ALIASES: Add Adam Piasecki (c4rt0)

### PR DESCRIPTION
**Pull request by c4rt0**
Modified content as instructed by provided documentation

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -131,6 +131,7 @@ aliases:
     - prestist
     - marmijo
     - RishabhSaini
+    - c4rt0
```
Comment (sadasu) [inline OWNERS_ALIASES:134]: Could you please add it in alphabetical order?

Comment (c4rt0) [reply OWNERS_ALIASES]: Thank you for this suggestion! Changes implemented.

**Pull request merged**